main.c

```diff
@@ -466,13 +475,41 @@ static void setup_property_changed_listener(void) {
 }
 #endif
 
+static void wl_seat_capabilities_handler(
```

Comment:
Maybe name this `seat_handle_capabilities`, for consistency with other functions? (Same applies to otehr function names)